### ISSUE TITLE
KernelExplainer shap values returning only zero

### ISSUE THREAD

**Issue by MarianneAK** [shap/shap#2926]
I am trying to apply the `KernelExplainer` on a complex `PyTorch` model. 
I have 5 inputs that I concatenated into a single vector (since I cannot pass multiple inputs to the `KernelExplainer`), which consist of 512 BERT tokens, and 34 other numerical features like so 
`explainer = shap.KernelExplainer(predictor, inputs.cpu().detach().numpy(), link='logit')`

Then I split these vectors back in my `predictor` callback function so that I can input them separately to each input layer.  I also have multiple outputs. 
But I am not sure why I keep obtaining only zero shap values. Could it be a regularization issue ? Or because my inputs are not normalized ? Since I am mixing up tokens with decimals. 
I tried many configurations in my `KernelExplainer` but nothing worked.

**Comment by paulsushmita** [comment]
Hi @MarianneAK. I ran into a similar issue of getting zeroes from Kernel explainer. I tried several things but was not successful. 

You seem to have closed the issue without other comments. Did you find a fix to this problem? If yes, can you please share how did you tackle the problem?
